gson/src/test/java/com/google/gson/functional/DefaultTypeAdaptersTest.java

```diff
@@ -710,6 +711,20 @@ public void testStringBufferDeserialization() {
     StringBuffer sb = gson.fromJson("'abc'", StringBuffer.class);
     assertThat(sb.toString()).isEqualTo("abc");
   }
+  @Test
+  public void testSetDateFormatWithInvalidPattern() {
+    GsonBuilder builder = new GsonBuilder();
+    String invalidPattern = "This is a invalid Pattern";
+    assertThrows(IllegalArgumentException.class, () -> {
+      builder.setDateFormat(invalidPattern);
+    });
+  }
+  @Test
```

Comment:
Just one tiny thing: could you add a blank line before each of these new test methods?

Reply:
OK, I am sorry about that.